firestore.rules

```diff
@@ -6,6 +6,10 @@ rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
 
+    match /{document=**} {
```

Comment:
this will only be used for testing purposes on the Firebase emulator, so that no one has ever admin access on the real database. Moreover, I will add a cloud function checking every night that no user has admin rights in second version PR